airbyte-featureflag/src/main/kotlin/Client.kt

```diff
@@ -118,7 +118,7 @@ class LaunchDarklyClient(private val client: LDClient) : FeatureFlagClient {
  *
  * @param [values] is a map of [Flag.key] to enabled/disabled status.
  */
-class TestClient @JvmOverloads constructor(val values: Map<String, Boolean> = mapOf()) : FeatureFlagClient {
+class TestClient @JvmOverloads constructor(val values: Map<String, Boolean> = mutableMapOf()) : FeatureFlagClient {
```

Comment:
what is this change for?

Reply:
This allows for easier testing with the feature flag `TestClient`.  Maps in kotlin default to immutable maps, you have to explicitly label them as `mutable`.

The use case for this test client is to allow you to modify the map within a test to verify behavior:
```java
@Test
public void testSomething() {
  final var ffClient = new TestClient();
  // test scenario A
  ffClient.values.add("flagA", true);
  // test scenario B
}
```